jujugui/static/gui/src/app/utils/environment-change-set.js

```diff
@@ -374,6 +375,7 @@ YUI.add('environment-change-set', function(Y) {
           window.setTimeout(this._commitNext.bind(this, env, currentIndex), 0);
         } else {
           this.currentLevel = -1;
+          this.committing = false;
```

Comment:
Nice, can we demonstrate that, sooner or later, in spite of errors, committing will be reset to false?
If so, could you please add a comment explaining why?